router.spec.js

```diff
@@ -9,7 +11,7 @@ const db = require('./db/db');
 const contribution = require('./db/models/contribution');
 const config = require('./config');
 
-const should = chai.should();
+const should = chai.should(); // eslint-disable-line no-unused-vars
```

Comment:
Please remove unused variables. The response of `chaiHttp` has a `should` method built in.